pipenv/cli.py

```diff
@@ -392,7 +392,7 @@ def do_create_virtualenv(three=None, python=None):
 
     # Actually create the virtualenv.
     with spinner():
-        c = delegator.run(cmd, block=False)
+        c = delegator.run(cmd, block=False, binary=False, timeout=PIPENV_TIMEOUT)
```

Comment:
No it doesn't I thought it was causing some tests to fail so I added it in, will remove!